Error: Failed to load plugin '@tybalt/eslint-plugin' declared in '.eslintrc': Cannot find module '@typescript-eslint/utils'

```sh
» npm run lint

> i-want-to-be-a-fan@1.0.0 lint
> eslint js/*.mjs


Oops! Something went wrong! :(

ESLint: 8.56.0

Error: Failed to load plugin '@tybalt/eslint-plugin' declared in '.eslintrc': Cannot find module '@typescript-eslint/utils'
Require stack:
- /Users/doug/code/i-want-to-be-a-fan/node_modules/@tybalt/eslint-plugin/dist/rules/component-names-are-multi-word.js
- /Users/doug/code/i-want-to-be-a-fan/node_modules/@tybalt/eslint-plugin/dist/rules/index.js
- /Users/doug/code/i-want-to-be-a-fan/node_modules/@tybalt/eslint-plugin/dist/index.js
- /Users/doug/code/i-want-to-be-a-fan/node_modules/@eslint/eslintrc/dist/eslintrc.cjs
Referenced from: /Users/doug/code/i-want-to-be-a-fan/.eslintrc
    at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
    at Module._load (node:internal/modules/cjs/loader:985:27)
    at Module.require (node:internal/modules/cjs/loader:1235:19)
    at require (node:internal/modules/helpers:176:18)
    at Object.<anonymous> (/Users/doug/code/i-want-to-be-a-fan/node_modules/@tybalt/eslint-plugin/dist/rules/component-names-are-multi-word.js:17:28)
    at Module._compile (node:internal/modules/cjs/loader:1376:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
    at Module.load (node:internal/modules/cjs/loader:1207:32)
    at Module._load (node:internal/modules/cjs/loader:1023:12)
    at Module.require (node:internal/modules/cjs/loader:1235:19)
```

--- a/packages/eslint-plugin/package.json
+++ b/packages/eslint-plugin/package.json
@@ -17,12 +17,14 @@
         "tybalt"
     ],
     "author": "Douglas Wade <douglas.b.wade@gmail.com>",
+    "dependencies": {
+        "@typescript-eslint/utils": "^6.11.0"
+    },
     "devDependencies": {
         "@types/node": "^20.10.7",
         "@typescript-eslint/eslint-plugin": "^6.7.4",
         "@typescript-eslint/parser": "^6.17.0",
         "@typescript-eslint/rule-tester": "^6.18.0",
-        "@typescript-eslint/utils": "^6.11.0",
         "esbuild": "^0.19.10",
         "esbuild-jest": "^0.5.0",
         "eslint": "^8.56.0",

